sycl/test/lit.cfg.py

```diff
@@ -75,9 +75,11 @@
 
 llvm_config.add_tool_substitutions(['llvm-spirv'], [config.sycl_tools_dir])
 
-backend=lit_config.params.get('SYCL_BE', "PI_OPENCL")
-lit_config.note("Backend (SYCL_BE): {}".format(backend))
-config.substitutions.append( ('%sycl_be', backend) )
+backend=lit_config.params.get('SYCL_BE', "opencl")
+lit_config.note("Backend: {}".format(backend))
+config.substitutions.append( ('%sycl_be', { 'opencl': 'PI_OPENCL',  'cuda': 'PI_CUDA', 'level_zero': 'PI_LEVEL_ZERO'}[backend]) )
+config.substitutions.append( ('%BE_RUN_PLACEHOLDER', "env SYCL_DEVICE_FILTER={SYCL_BE} ".format(SYCL_BE=backend)) )
+config.substitutions.append( ('%HOST_RUN_PLACEHOLDER', "env SYCL_DEVICE_FILTER=host ") )
```

Comment:
According to my understanding `_PLACEHOLDER` suffix indicates that if particular back-end is not enabled, the command won't be executed. `host` back-end is always enabled, so using this suffix doesn't make much sense to me.
How about `HOST_RUN_PLACEHOLDER` -> `RUN_ON_HOST`?

Did you consider removing substitution variable at all? I think it might be possible as we moved all tests executed on non-host back-ends to a separate directory. 
I'm not sure if it's feasible, but we could implicitly set `env SYCL_DEVICE_FILTER=host` for all the tests and use explicit substitution variables for "on-device" tests to override the default value of `SYCL_DEVICE_FILTER` variable. If you decide to apply this, please, make sure that `SYCL_DEVICE_FILTER=host` is added to the run command to make test results easy to reproduce manually.

Reply:
Rename substituition from HOST_RUN_PLACEHOLDER  to RUN_ON_HOST
Implicit env SYCL_DEVICE_FILTER=host may confuse users. The substitution itself is preserved as it makes tests more straightforward